keanu-python/keanu/vertex.py

```diff
@@ -122,8 +123,22 @@ def __hash__(self):
         return hash(self.get_id())
 
     def observe(self, v):
-        from keanu.tensor import Tensor
-        self.unwrap().observe(Tensor(v).unwrap())
+        self.unwrap().observe(kn.Tensor(v).unwrap())
+
+    def set_value(self, v):
+        self.unwrap().setValue(kn.Tensor(v).unwrap())
+
+    def set_and_cascade(self, v):
+        self.unwrap().setAndCascade(kn.Tensor(v).unwrap())
+
+    def sample(self):
+        return self.__to_np_array(self.unwrap().sample())
+
+    def get_value(self):
+        return self.__to_np_array(self.unwrap().getValue())
+
+    def __to_np_array(self, value):
```

Comment:
Should probably now use/remove the `_to_np_array` in `tensor.py`.

Reply:
good call - done